src/test/java/com/pba/authservice/unit/ActiveUserDaoUnitTest.java

```diff
@@ -0,0 +1,179 @@
+package com.pba.authservice.unit;
+
+import com.pba.authservice.exceptions.AuthDaoException;
+import com.pba.authservice.mockgenerators.ActiveUserMockGenerator;
+import com.pba.authservice.persistance.model.ActiveUser;
+import com.pba.authservice.persistance.repository.ActiveUserDao;
+import com.pba.authservice.persistance.repository.mappers.ActiveUserRowMapper;
+import com.pba.authservice.persistance.repository.sql.ActiveUserSqlProvider;
+import org.checkerframework.checker.units.qual.A;
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class ActiveUserDaoUnitTest {
+    @InjectMocks
+    private ActiveUserDao activeUserDao;
+
+    @Mock
+    private ActiveUserSqlProvider activeUserSqlProvider;
+
+    @Mock
+    private ActiveUserRowMapper activeUserRowMapper;
+
+    @Mock
+    private JdbcTemplate jdbcTemplate;
+
+    private ActiveUserMockGenerator activeUserMockGenerator;
```

Comment:
make ActiveUserMockGenerato's methods static, no need to create instance. Same above.